app/lib/features/chat_ng/providers/notifiers/reply_messages_notifier.dart

```diff
@@ -0,0 +1,66 @@
+import 'package:acter/common/providers/chat_providers.dart';
+import 'package:acter/features/chat_ng/models/reply_message_state.dart';
+import 'package:acter/features/chat_ng/providers/chat_room_messages_provider.dart';
+import 'package:flutter_riverpod/flutter_riverpod.dart';
+import 'package:logging/logging.dart';
+
+final _log = Logger('a3::chat::reply_notifier');
+
+class ReplyMessageNotifier
+    extends AutoDisposeFamilyAsyncNotifier<ReplyMsgState, ReplyMsgInfo> {
+  Future<void> retryLoad() async {
+    state = const AsyncValue.loading();
+    state = await AsyncValue.guard(() => build(arg));
```

Comment:
guard crates a loading around the inner fn ...
```suggestion
    state = AsyncValue.guard(() => build(arg));
```